File: 3st/Juram/programmer/모음 제거.js
function solution(my_string) {
    var answer = '';
    for(let i=0; i<my_string.length; i++){
        switch(my_string[i]){
            case "a": case "e": case "i": case "o": case "u":
                continue;
            default:answer+=my_string[i]
        }
        
    }
    return answer;
}

File: 4st/Seohyun/programmer/모음 제거.java
class Solution {
    public String solution(String my_string) {
        String[] aeiou = {"a", "e", "i", "o", "u"};
        String answer = my_string;
        
        for(int i=0; i<aeiou.length; i++) {
            answer = answer.replace(aeiou[i], "");
        }
        return answer;
    }
}